tests/integration.rs

```diff
@@ -9136,6 +9136,40 @@ mod vfio {
 
         handle_child_output(r, &output);
     }
+
+    #[test]
+    fn test_nvidia_card_iommu_address_width() {
+        let jammy = UbuntuDiskConfig::new(JAMMY_VFIO_IMAGE_NAME.to_string());
+        let guest = Guest::new(Box::new(jammy));
+        let api_socket = temp_api_path(&guest.tmp_dir);
+
+        let mut child = GuestCommand::new(&guest)
+            .args(["--cpus", "boot=4"])
+            .args(["--memory", "size=4G"])
+            .args(["--kernel", fw_path(FwType::RustHypervisorFirmware).as_str()])
+            .args(["--device", format!("path={NVIDIA_VFIO_DEVICE}").as_str()])
+            .args(["--platform", "iommu_address_width=42"])
```

Comment:
I don't think this test will have a `virtio-iommu` device as that is only added if any device has `iommu=on`

Reply:
Good catch!

Should I keep the test, modifying it to use `iommu=on`, or have it somewhere else where it's more appropriate in the test suite?

Reply:
Added `iommu=on`, but not sure if the host running this test is actually configured to support this option.